src/jenkins-enhanced.coffee

```diff
@@ -353,13 +486,86 @@ class HubotJenkinsPlugin extends HubotMessenger
     "HEALTH: #{result}\n"
 
   _getJob: (escape = false) =>
-    job = @msg.match[1].trim()
-
-    # if the provided name is an alias, provide it's mapped job name
-    aliases = @_getSavedAliases()
-    job     = aliases[job] if aliases[job]
-
-    if escape then @_querystring.escape(job) else job
+    # check if its a job we already stored
```

Comment:
Consider breaking the `_getJob` method into separate smaller methods with descriptive names to reduce the nesting of these if/else statements.